apps/web/src/components/SearchRequestTable/components/SearchRequestsTableFilterDialog.tsx

```diff
@@ -0,0 +1,180 @@
+  onOpenChange: (open: boolean) => void;
+  onSubmit: SubmitHandler<FormValues>;
+  activeFilters: FormValues;
+}
+
+export const SearchRequestsTableFilterDialog = ({
+  onSubmit,
+  activeFilters,
+  isOpen,
+  onOpenChange,
+}: SearchRequestsTableFilterDialogProps): JSX.Element => {
+  const { formatMessage } = useIntl();
+  const { optionsData, rawGraphqlResults } = useFilterOptions();
+
+  return (
+    <Dialog.Root open={isOpen} onOpenChange={onOpenChange}>
+      <Dialog.Trigger>
+        <Button color="secondary" type="button" icon={AdjustmentsVerticalIcon}>
+          {formatMessage({
+            defaultMessage: "Filters",
+            id: "1HPhji",
+            description:
+              "Text label for button to open filter dialog on admin tables.",
+          })}
+        </Button>
+      </Dialog.Trigger>
+      <Dialog.Content>
+        <Dialog.Header
+          subtitle={formatMessage({
+            defaultMessage:
+              "Narrow down your table results using the following filters.",
+            id: "hqZfyb",
+            description: "Candidate search filter dialog: subtitle",
+          })}
+        >
+          {formatMessage({
+            defaultMessage: "Select filters",
+            id: "P9SZBZ",
+            description: "Candidate search filter dialog: title",
+          })}
+        </Dialog.Header>
+        <Dialog.Body>
+          <BasicForm
+            {...{ onSubmit }}
+            options={{
+              defaultValues: activeFilters,
+            }}
+          >
+            <div data-h2-flex-grid="base(flex-start, x1, x.5)">
+              <div data-h2-flex-item="base(1of1) p-tablet(1of2) laptop(3of5)">
+                <MultiSelectField
+                  id="status"
+                  label={formatMessage(adminMessages.status)}
+                  options={optionsData.status}
+                />
+              </div>
+              <div data-h2-flex-item="base(1of1) p-tablet(1of2) laptop(1of3)">
+                <MultiSelectField
+                  id="departments"
+                  label={formatMessage(adminMessages.departments)}
+                  options={optionsData.departments}
+                  isLoading={rawGraphqlResults.departments.fetching}
+                />
+              </div>
+              <div data-h2-flex-item="base(1of1) p-tablet(1of2) laptop(1of3)">
+                <MultiSelectField
+                  id="classifications"
+                  label={formatMessage(adminMessages.classifications)}
+                  options={optionsData.classifications}
+                  isLoading={rawGraphqlResults.classifications.fetching}
+                />
+              </div>
+              <div data-h2-flex-item="base(1of1) p-tablet(1of2) laptop(3of5)">
+                <MultiSelectField
+                  id="streams"
+                  label={formatMessage(adminMessages.streams)}
+                  options={optionsData.streams}
+                />
+              </div>
+            </div>
```

Comment:
In the screenshot of the ticket, these are all 50% in a grid. We could simplify this and sync up with that by use `display: grid;`

```tsx
<div
    data-h2-display="base(grid)"
    data-h2-grid-template-columns="p-tablet(repeat(2, 1fr))"
    data-h2-gap="base(x1)"
>
  <input />
  <input />
  <input />
  <input />
</div>
```

Reply:
Yes, that seems to work quite nicely thanks
https://github.com/GCTC-NTGC/gc-digital-talent/pull/6897/commits/a5d001e8212135f8f4257cfc755e1758a568ba03